<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticBrightHistogramFormClass</class>
 <widget class="QDialog" name="StatisticBrightHistogramFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StatisticBrightHistogramForm</string>
  </property>
  <widget class="QPushButton" name="ButtonClose">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>閉じる</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyStatisticBlock.qrc">
     <normaloff>:/Qtres/Close.png</normaloff>:/Qtres/Close.png</iconset>
   </property>
  </widget>
  <widget class="QSpinBox" name="EditPickupL">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QFrame" name="frameLineGraph">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="EditPickupH">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>抽出輝度　明側</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="radioButtonDarkSide1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLightSide1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Light</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButtonReflect">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>反映</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyStatisticBlock.qrc">
     <normaloff>:/Qtres/Reflection.png</normaloff>:/Qtres/Reflection.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>抽出輝度　暗側</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PropertyStatisticBlock.qrc"/>
 </resources>
 <connections/>
</ui>
